src/components/Shop/Modal/index.tsx

```diff
@@ -0,0 +1,79 @@
+import React, { useContext, FC } from "react";
```

Comment:
As we might be using the modal more than once, it should only render a children prop that is a ReactNode, perhaps a title, and the back arrow.